.eslintrc.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   root: true,
+  parser: 'babel-eslint',
```

Comment:
I did not add this manually, unsure how this came in. @rwjblue any ideas?

Reply:
Rebase issue maybe? I don’t think we should need a special parser should be able to remove...